src/test/regress/expected/failure_real_time_select.out

```diff
@@ -125,27 +131,23 @@ results AS (SELECT * FROM test_table)
 SELECT * FROM test_table
 WHERE test_table.id > (SELECT id FROM results);
 ERROR:  canceling statement due to user request
--- Since Citus opens a new connection after a failure within the real time
--- execution and after(1).kill() kills connection after a successful execution
--- for each connection, following transaciton does not fail.
+-- Citus fails if the connection that is already used fails afterwards
```

Comment:
This is a behavior change we implemented in the unified executor. If any query on a shard over a connection succeeds, further failures end with errors. See here for a more detailed [discussion](https://github.com/citusdata/citus/pull/2713#issue-279801940)